ca-rotation-script/script/manual-CA-Rotation.sh

```diff
@@ -242,56 +239,56 @@ openssl req -new -key "${FRONT_PROXY_CLIENT_KEY}" -out /tmp/temp.csr -subj "/CN=
 openssl x509 -req -in /tmp/temp.csr -CA "${FRONT_CA_CRT}" -CAkey "${FRONT_CA_KEY}" -CAcreateserial -out "${FRONT_PROXY_CLIENT_CRT}" -days 825 -sha256 
 
 # Copy new certs to /etc/kubernetes/pki/
-docker cp cert/apiserver/. ${KUBE_CONTROLLER_MANAGER}:/etc/kubernetes/pki/
-docker cp cert/frontendProxyCA/. ${KUBE_CONTROLLER_MANAGER}:/etc/kubernetes/pki/
+docker cp cert/apiserver/. "${KUBE_CONTROLLER_MANAGER}":/etc/kubernetes/pki/
+docker cp cert/frontendProxyCA/. "${KUBE_CONTROLLER_MANAGER}":/etc/kubernetes/pki/
 
 # Restart apiserver and scheduler
 sleep 20
-kubectl ${WORKLOAD_CLUSTER} -n kube-system delete pod kube-apiserver-${KUBE_CONTROLLER_MANAGER} || true
-kubectl ${WORKLOAD_CLUSTER} -n kube-system delete pod kube-scheduler-${KUBE_CONTROLLER_MANAGER} || true
+kubectl ${WORKLOAD_CLUSTER} -n kube-system delete pod kube-apiserver-"${KUBE_CONTROLLER_MANAGER}" || true
+kubectl ${WORKLOAD_CLUSTER} -n kube-system delete pod kube-scheduler-"${KUBE_CONTROLLER_MANAGER}" || true
 
 # Step 8.4: Annotate any Daemonsets and Deployments to trigger pod replacement in a safer rolling fashion
 WORKLOAD_CLUSTER="--kubeconfig=capi-kubeconfig"
```

Comment:
Not really part of the change, but instead of defining `${WORKLOAD_CLUSTER}` in each `kubectl` call, wouldn'it be easier to `export KUBE_CONFIG=capi-kubeconfig` and have it applied automatically?

Reply:
perhaps, I will leave it as it is, since I am only fixing shellchecks here.